### PR TITLE
[No QA] Show Workspace Chat first instead of opening Global Create

### PR DESCRIPTION
**Pull request by luacmartins**
### Details
Updates the welcome actions to show the Workspace chat first instead of opening the global create.

cc @TomatoToaster 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/195473

### Tests
1. Get the `accountID` and `reportID` for an existing account and chat.
2. Update the `chatType` of that chat to `policyExpenseChat` by running the following query: `UPDATE reportNameValuePairs SET value = 'policyExpenseChat' WHERE name = 'chatType' AND reportID = <reportID>;`
3. Update the `isFirstTimeNewExpensifyUser` for that user to true by running the following query `UPDATE nameValuePairs SET value = true WHERE name = 'isFirstTimeNewExpensifyUser' AND accountID = <accountID>;`
4. Clear browser data
5. Log into that account and notice that the chat switches to the `reportID` chat.
6. Create a new account by running `./script/clitools.sh generator:account`
7. Login and notice that the global create menu opens.

- [X] Verify that no errors appear in the JS console

### QA Steps
None, as Workspace chats are not available yet.

- [ ] Verify that no errors appear in the JS console

### Tested On

- [X] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/22219519/155311082-a232b022-ccf2-461c-a76b-b56b7ae34b70.mov

#### Mobile Web

https://user-images.githubusercontent.com/22219519/155311114-e2d54dca-45a6-4ccf-bbaa-bddfbff95c40.mov

#### Desktop

https://user-images.githubusercontent.com/22219519/155311129-7550fd5c-9cf4-47ef-ba84-f1742a700d85.mov

#### iOS

https://user-images.githubusercontent.com/22219519/155311165-8202bf6c-ba0a-4c1e-87b4-7adbdee650a6.mov

#### Android

https://user-images.githubusercontent.com/22219519/155311187-d8edfb74-cf14-4f17-bd28-12e39048da62.mov

### CODE REVIEW

File: src/pages/home/HeaderView.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const HeaderView = (props) => {
                         </Pressable>
                     </Tooltip>
                 )}
-                {props.report && props.report.reportName && (
+                {Boolean(props.report && props.report.reportName) && (
```
Comment (luacmartins) [inline src/pages/home/HeaderView.js:116]: I added this to fix a RN error that I was seeing where text is a child of View

**Comment by MonilBhavsar** [comment]
Can you please add test screenshots

**Review by MonilBhavsar — APPROVED** [review]
Simple change and looks good to me
@TomatoToaster could you please give this a quick review

File: src/pages/home/HeaderView.js (hunk 1 above)
Comment (TomatoToaster) [reply src/pages/home/HeaderView.js:116]: I haven't seen it done like this, good thinking.

**Review by TomatoToaster — APPROVED** [review]
Niceeee

**Pull request merged**